common/lib/capa/capa/templates/jstextline.html

```diff
@@ -1,3 +1,4 @@
+<%! from openedx.core.djangolib.markup import HTML %>
```

Comment:
@alisan617 please just delete this file as part of this PR (it doesn't impact any other parts in the code).